src/components/aria-live/aria-live.html

```diff
@@ -0,0 +1,2 @@
+<ng-content></ng-content>
```

Comment:
Oh, if it has no `ng-content`, then it can't be a parent component. That would mean, we are not able to inject the component. 

Otherwise we need to provide a global OpaqueToken or something else. 
In my opinion, injecting it as the parent component is very elegant.

What are your thoughts?